### DISCUSSION TITLE
Place field does not store additional fields

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#3629] [Q&A] [answered]
- Laravel Version: 8.7.0
- Nova Version: 3.3.0
- PHP Version: 8.0
- Database Driver & Version  MySQL
- Browser type and version: Latest Chrome & Firefox or Safari

### Description:

I have a Place field within my application and would like to store latitude & longitude based on the Place Input. When entering a place, the autocomplete for latitude & longitude works. But after updating the form, the values are empty, plus no entries are saved within the database.

<img width="1363" alt="grafik" src="https://user-images.githubusercontent.com/10268813/141298693-dc781c5a-4ae9-47fb-b632-ed06a530a71b.png">

<img width="1373" alt="grafik" src="https://user-images.githubusercontent.com/10268813/141298884-c07f35c1-706f-4d41-abaf-e9d920ceb239.png">

### Detailed steps to reproduce the issue on a fresh Nova installation:

**Add the following database schema to a model**

<img width="662" alt="grafik" src="https://user-images.githubusercontent.com/10268813/141298522-d07ac598-469c-4e15-ae5e-b9f6c26b1770.png">

**Create a Laravel Nova Place field within your Model Resource**

```
Place::make('Place','place')
->onlyCities()
->latitude('latitude')
->longitude('longitude'),

Text::make('Latitude','latitude')
->readonly()
->hideFromIndex(),

Text::make('Longitude','longitude')
->readonly()
->hideFromIndex(),

```

**Add decimal casts to your model**

```
protected $casts = [
'latitude' => 'decimal',
'longitude' => 'decimal'
];
```

**Comment by crynobone** [comment, marked as answer]
Laravel Nova doesn't send `readonly` field when submitting the form, maybe try using `Hidden` field?

**Reply by StanBarrows** [reply]
Thank you for the quick answer. Readonly & OnlyOnDetails or HideWhenCreating/Updating worked not in my case.
